migrator/config-loader.lua

```diff
@@ -0,0 +1,60 @@
+local fio = require('fio')
+local checks = require('checks')
+local log = require('log')
+
+local Loader = {}
+Loader.__index = Loader
+
+local function __must_sort(m)
+    table.sort(m, function(a, b) return a.name < b.name end)
+end
+
+local function assert_migration(migration)
+    checks({
+        name = 'string',
+        up = 'function'
+    })
+    return migration
+end
+
+function Loader:list()
+    local ca = require("cartridge.confapplier")
+    local cfg = ca.get_active_config():get_plaintext()
+    if cfg == nil then
+        return {}
+    end
+
+    local result = {}
+
+    for k, v in pairs(cfg) do
+        if fio.dirname(k) == self.dir_name then
```

Comment:
I don't get it - why would you need it?

Reply:
Probably if you'd expand readme I'd get it better

Reply:
clusterwide config does not stores directories hierarchically, it's a map with keys like "migration/source/0001.lua" so to get migrations one should filter out all other keys, changed filter to match string with "startswith"